<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fotoğraflı HES Kodu Sorgulama</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="resim">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>150</width>
      <height>200</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Desktop/resim1.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="hata">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>215</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color : red}</string>
    </property>
    <property name="text">
     <string>Hata</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>231</x>
      <y>41</y>
      <width>215</width>
      <height>250</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>HES Kodu    :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="hesKodu"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>TC Kimlik No:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Durum         :</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="durum">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="kullaniciAdi">
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="sifre">
       <property name="inputMethodHints">
        <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelKullaniciAdi">
       <property name="text">
        <string>Kullanıcı adı  :</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelSifre">
       <property name="text">
        <string>Şifre            :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="veritabanindan">
       <property name="text">
        <string>Veritabanından</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="eDevletten">
       <property name="text">
        <string>E-Devletten</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QPushButton" name="sorgula">
       <property name="text">
        <string>Sorgula</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QPushButton" name="temizle">
       <property name="text">
        <string>Temizle</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>HES Kodu     :</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="labelHesKodu">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="tcKimlikNo">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDosya">
    <property name="title">
     <string>Dosya</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuKay_t">
    <property name="title">
     <string>Kayıt</string>
    </property>
    <addaction name="actionEkle"/>
    <addaction name="actionGuncelle"/>
    <addaction name="actionSil"/>
   </widget>
   <widget class="QMenu" name="menuYardim">
    <property name="title">
     <string>Yardım</string>
    </property>
   </widget>
   <addaction name="menuDosya"/>
   <addaction name="menuKay_t"/>
   <addaction name="menuYardim"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEkle">
   <property name="text">
    <string>Ekle</string>
   </property>
  </action>
  <action name="actionGuncelle">
   <property name="text">
    <string>Güncelle</string>
   </property>
  </action>
  <action name="actionSil">
   <property name="text">
    <string>Sil</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
